apps/stix-shifter/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM debian:buster-slim
+
+RUN apt-get -qq update && apt-get -qqy install \
+  python3-pip software-properties-common
+
+RUN pip3 install --upgrade pip
+
+WORKDIR /opt/tenzir/threatbus/stix-shifter-threatbus
```

Comment:
For the VAST Dockerfile, we use `/usr/local` as prefix. Should we be using that here, too?

Reply:
As discussed via call, that's not needed due to using the ENTRYPOINT and CMD.